development/webpack/utils/plugins/LavamoatPlugin/index.ts

```diff
@@ -0,0 +1,126 @@
+import { join } from 'node:path';
+import LavamoatPlugin from '../../../../../../LavaMoat/packages/webpack/src/plugin.js';
+import type { Args } from '../../cli';
+
+export const lavamoatPlugin = (args: Args) =>
+  new LavamoatPlugin({
+    rootDir: join(__dirname, '../../../../../'), // While ../../../../../app is the main dir for the webpack build to use as context, the project root where package.json is one level up. This discrepancy needs to be explained to LavaMoat plugin as it's searching for the package.json in the compilator.context by default.
+    diagnosticsVerbosity: 0,
+    generatePolicyOnly: args.generatePolicy,
+    runChecks: true, // Candidate to disable later for performance. useful in debugging invalid JS errors, but unless the audit proves me wrong this is probably not improving security.
+    readableResourceIds: true,
+    inlineLockdown: /^runtime|contentscript\.js/u,
+    debugRuntime: args.lavamoatDebug,
+    lockdown: {
+      consoleTaming: 'unsafe',
+      errorTaming: 'unsafe',
+      stackFiltering: 'verbose',
+      overrideTaming: 'severe',
+      localeTaming: 'unsafe',
+      errorTrapping: 'none',
+      reporting: 'none',
+    },
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    runtimeConfigurationPerChunk_experimental: (chunk) => {
+      switch (chunk.name) {
+        case 'scripts/contentscript.js':
+          return {
+            mode: 'safe',
+            embeddedOptions: {
+              scuttleGlobalThis: {
+                enabled: true,
+                // Globals used by the contentscript
+                exceptions: ['browser', 'chrome', 'btoa'],
+              },
+            },
+          };
+        case 'scripts/inpage.js':
+          return { mode: 'unlocked_unsafe' };
+        case 'bootstrap':
+          return { mode: 'unlocked_unsafe' };
+        case 'runtime':
+          return {
+            mode: 'safe',
+            // If snow is enabled, it needs to run outside of LavaMoat
+            staticShims: args.snow
+              ? [
+                  join(
+                    __dirname,
+                    '../../../../../node_modules/@lavamoat/snow/snow.prod.js',
+                  ),
```

Comment:
Something like `require.resolve('@lavamoat/snow/snow.prod.js')` should work here